packages/components/inputs/text-input/src/text-input.story.js

```diff
@@ -17,7 +17,16 @@ storiesOf('Components|Inputs', module)
   })
   .add('TextInput', () => {
     return (
-      <Section>
+      <Section
```

Comment:
This allows to change the background color of the story container to check the correct behaviour.